File: frontend/src/app/dtos/event/event-search-result.dto.ts
export class EventSearchResultDto {
    constructor(
      public id: number,
      public title: string,
      public description: string,
      public categoryId: number,
      public soldOut: boolean,
      public hasShowingsInFuture: boolean,
      public imageFile: string,
      public durationHours: number,
      public durationMinutes: number
    ) {
    }
  }

File: backend/src/main/java/at/ac/tuwien/sepm/groupphase/backend/endpoint/dto/event/EventSearchResultDto.java
package at.ac.tuwien.sepm.groupphase.backend.endpoint.dto.event;

public record EventSearchResultDto(
    Long id,
    String title,
    String description,
    Long categoryId,
    Boolean soldOut,
    Boolean hasShowingsInFuture,
    String imageFile,
    Long durationHours,
    int durationMinutes) {}
